sequencing/ATACSeq/jobSubmission/0_setUp.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+#SBATCH --export=ALL # export all environment variables to the batch job.
+#SBATCH -p mrcq # submit to the serial queue
+#SBATCH --time=48:00:00 # Maximum wall time for the job.
+#SBATCH -A Research_Project-MRC190311 # research project to submit under. 
+#SBATCH --nodes=1 # specify number of nodes.
+#SBATCH --ntasks-per-node=16 # specify number of processors per node
+#SBATCH --mail-type=END # send email at job completion 
+#SBATCH --output=ATACSetUpS0-%A.log
+#SBATCH --error=ATACSetUpS0-%A.err
+#SBATCH --job-name=ATACSetUpS0
+
+## ==========================================================================================================##
+##                               ATAC-seq pipeline STEP 0: Set up                                            ##
+## ==========================================================================================================##
+## EXECUTION: sbatch ./sequencing/ATACSeq/jobSubmission/0_setUp.sh <project name>                            ||
+## - execute from scripts directory                                                                          ||
+##                                                                                                           ||
+## INPUTS:                                                                                                   || 
+## $1 -> <project name> name of project to set up pipeline for.                                              || 
+##                                                                                                           ||
+## DESCRIPTION: This script sets up the required directories, libraries and modules to run the ATAC-seq      || 
+## pipeline. Will check the config file for the project and the initial paths required for the first steps of|| 
+## the analysis.                                                                                             ||
+##                                                                                                           ||
+## REQUIRES:                                                                                                 ||
+## - File in ${METADIR}/samples.txt that lists sample names.                                                 ||
+## ==========================================================================================================##
+
+
+## ============ ##
+##    SET UP    ##
+## ============ ##
+
+## print start date and time
+echo Job started on:
+date -u
+
+
+LOG_DIR=ATACSeq/logFiles/${USER}/${SLURM_JOB_ID}
+echo "Log files will be moved to dir: " $LOG_DIR
+mkdir -p $LOG_DIR
+mv ATACSetUpS0-${SLURM_JOB_ID}* $LOG_DIR
+
+## ============ ##
+##    CHECKS    ##
+## ============ ##
+
+## CONFIG FILE ##
+
+## Check if config file for input project exist
+
+if [[ -s "/lustre/projects/Research_Project-MRC190311/ATACSeq/$1/config.txt" ]]
+then
+  source "/lustre/projects/Research_Project-MRC190311/ATACSeq/$1/config.txt"
```

Comment:
Need another opinion on this one. But I'd say instead of forcing the config file to be in the root of the project directory. You could use the find command to check for the existence of the config file. Then you could use the output of the find command to source the file. This would make the placement of the config file less strict (some users might want to put the config files in a separate directory).